projects/ffmpeg/build.sh

```diff
@@ -109,8 +125,8 @@ make install
 
 cd $SRC/libxml2
 ./autogen.sh --prefix="$FFMPEG_DEPS_PATH" --enable-static \
-    --without-debug --without-ftp --without-http \
-    --without-legacy --without-python
+	--without-debug --without-ftp --without-http \
+	--without-legacy --without-python
```

Comment:
No need to change indention, nothing else is changed in t these lines i think.
If indention needs some cleanup that should be in a separate commit